### ISSUE TITLE
Incorrect national formatting for Canadian numbers

### ISSUE THREAD

**Issue by philippevezina** [daddyz/phonelib#240]
Formatting for Canadian phone numbers is incorrect:
```
Phonelib.parse('418 928 3648', 'CA').national
=> "418 928 3648"
```

What should be returned: `(418) 928-3648`.
Here is the proof of concept with `libphonenumber`'s sandbox: http://libphonenumber.appspot.com/phonenumberparser?number=4189283648&country=CA&geocodingLocale=en-US

I can't say whether this is simply because it would have been fixed recently in `libphonenumber` and that `phonelib`'s data is outdated or if it is rather a bug with `phonelib`.

Documentation on how to update the data could be appropriate to include in the README or Wiki.

**Comment by philippevezina** [comment]
I see that the data has been updated since and the problem is still there. I would be inclined to say the problem is in the gem.

**Comment by daddyz** [comment]
@philippevezina thanks for reporting.
I found a bug in formatting rules. Fixed in version 0.6.57
```ruby
Phonelib.parse('418 928 3648', 'CA').national # => "(418) 928-3648"
```

**Comment by philippevezina** [comment]
That's great! Thank you very much!
